app.js

```diff
@@ -12,28 +12,34 @@
     ]
 };
 
-
 // Modal elements
 const modal = document.getElementById('modal');
 const submitNameBtn = document.getElementById('submit-name-btn');
 const inputName = document.getElementById('given-user-name');
 const errorMsg = document.getElementById('error-msg');
 
 // Game elements
-const paperButton = document.getElementById('paper');
-const rockButton = document.getElementById('rock');
-const scissorsButton = document.getElementById('scissors');
+const playerElements = {
+    paperButton: document.getElementById('paper'),
+    rockButton: document.getElementById('rock'),
+    scissorsButton: document.getElementById('scissors')
+}
 
-const chosenPaperElement = document.getElementById('chosen-paper');
-const chosenRockElement = document.getElementById('chosen-rock');
-const chosenScissorsElement = document.getElementById('chosen-scissors');
+// Chosen by user
+const chosen = {
```

Comment:
look - if you will name it like `playerChosenElements` it will be so descriptive, that comment could be removed and everyone understand what is going on. 
What is more - if you did an object (so variables inside it are extracted and there is no conflicts), you don't need this `chosen` prefix in elements (so - you could do `paperElement`. 
Look - you could use it like:
`playerChosenElements.paperElement` - you read it and you know that this is a paper element from player chosen elements. 
In general - it is *very important*:
- code should be as readable and understandable as comments would be totally useless (ofc, there could be exceptions)